.github/workflows/crates-builder.yaml

```
@@ -0,0 +1,59 @@
+      ref:
+        required: false
+        type: string
+        default: ""
+        description: "Git ref to checkout (defaults to GitHub's default behavior if empty)"
+
+jobs:
+  setup-builder:
+    name: Setup Rust & Cache
+    runs-on: ubuntu-latest
+    outputs:
+      cache-hit: ${{ steps.rust-cache.outputs.cache-hit }}
```

actionlint (1.7.12)

[error] property "cache-hit" is not defined in object type {}

(expression)